src/remarx/quotation/find_quotes.py

```diff
@@ -90,10 +133,44 @@ def main() -> None:
     log_level = logging.DEBUG if args.verbose else logging.INFO
     configure_logging(sys.stdout, log_level=log_level)
 
+    if len(args.paths) < 2:
+        parser.error("reuse corpus and output path are required")
+
+    if len(args.paths) > 3:
+        _error_exit(
+            "Error: specify at most one original corpus (file or directory) followed by reuse corpus and output path"
+        )
+
+    positional_paths = [pathlib.Path(p) for p in args.paths]
+    if len(positional_paths) == 2:
+        # When no original path is provided, fall back to the default directory
+        reuse_corpus, output_path = positional_paths
+        logger.info(
+            "No original corpora specified; defaulting to %s",
+            DEFAULT_ORIGINAL_CORPUS_DIR,
+        )
+        original_inputs = [DEFAULT_ORIGINAL_CORPUS_DIR]
+    else:
+        original_inputs = [positional_paths[0]]
+        reuse_corpus = positional_paths[1]
+        output_path = positional_paths[2]
```

Comment:
I don't know if it's a marimo sync error or if I'm misunderstanding your intention, but I couldn't specify multiple input paths.  However, using a directory did work.

Reply:
@tanhaow that was my original intent! I must have misread your code - I thought you were intentionally make it more flexible than that. Either way is fine with me - first option is simpler, but second option is useful if you've already implemented it.